flixel/input/touch/FlxTouch.hx

```diff
@@ -23,13 +23,25 @@ class FlxTouch extends FlxPointer implements IFlxDestroyable implements IFlxInpu
 	 */
 	public var touchPointID(get, never):Int;
 
+	/**
+	 * A value between 0.0 and 1.0 indicating force of the contact with the device. If the device does not support detecting the pressure, the value is 1.0.
+	 */
+	public var pressure(get, never):Float;
+
+	/**
+	 * Returns `true` if the touch event is caused by a stylus/pen device, rather than a finger touch.
+	 */
+	public var isPen(get, never):Bool;
```

Comment:
I'm not seeing the point of isPen, and I also think it's not doing what you claim. devices can have pressure sensitivity with fingers. we should just remove this. we can still set pressures of 0 to 1 if we think 0 is a mistake, but that has nothing to do with pen vs finger